src/Cloud/weglide/WeGlideSettings.hpp

```diff
@@ -0,0 +1,75 @@
+/*
+Copyright_License {
+
+  XCSoar Glide Computer - http://www.xcsoar.org/
+  Copyright (C) 2000-2021 The XCSoar Project
+  A detailed list of copyright holders can be found in the file "AUTHORS".
+
+  This program is free software; you can redistribute it and/or
+  modify it under the terms of the GNU General Public License
+  as published by the Free Software Foundation; either version 2
+  of the License, or (at your option) any later version.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with this program; if not, write to the Free Software
+  Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
+}
+*/
+
+#ifndef XCSOAR_CLOUD_WEGLIDE_WEGLIDESETTINGS_HPP
+#define XCSOAR_CLOUD_WEGLIDE_WEGLIDESETTINGS_HPP
+
+#include "util/StaticString.hxx"
+#include "util/TriState.hpp"
+#include "time/BrokenDate.hpp"
+
+#include <cstdint>
+#include <tchar.h>
+
+#define WEGLIDE_PILOT_ID_AS_TEXT  0
+
+/**
+ * WeGlide settings
+ */
+struct WeGlideSettings {
+  /**WeGlide communication
+   */
+  TriState enabled;
+
+  /**
+   * Automatic upload IGC file to WeGlide server after download from logger
+   */
+  bool automatic_upload;
+
+  // hard coded yet, maybe WeGlide change this in the future, then you have to
+  // make it setable!
+  // The documentation of the WeGlide API you can find:
+  // https://api.weglide.org/docs
```

Comment:
Bad comment style. For API documentation, use doxygen, and for general multi-line comments, use `/* */`. `//` is for single-line comments.

Reply:
Done